dist/scripts/check-shims-revisions.sh

```diff
@@ -35,15 +35,14 @@ function check-shims-revisions() {
     if [[ -f "$shim_version_path" ]] ; then
       curr_revision=$(grep "revision=" "${shim_version_path}" | cut -d'=' -f2)
       if [ -n "$pre_revision" ] && [[ "$curr_revision" != "$pre_revision" ]] ; then
-        echo
-        echo "Check Failed: git revisions between shims are not equal"
-        echo "Please check the revisions of each shim to see which one is inconsistent. Note, if building with Databricks those jars are built separately."
+        echo >&2 "Check Failed: git revisions between shims are not equal"
```

Comment:
Missing the original empty `echo`, who will print a space line.

Reply:
It's a empty line for show,  can be remove.

Reply:
Sure, I'll update it.